dvc/repo/get.py

```diff
@@ -33,6 +45,12 @@ def get(url, path, out=None, rev=None):
     tmp_dir = os.path.join(dpath, "." + str(shortuuid.uuid()))
     try:
         with external_repo(cache_dir=tmp_dir, url=url, rev=rev) as repo:
+            full_path = os.path.join(repo.root_dir, path)
+            path_in_repo = path_isin(full_path, repo.root_dir)
+            if os.path.exists(full_path) and path_in_repo:
```

Comment:
We may just check that path is not absolute. Absolute might only be an external out, not git tracked, so we skip here.

Reply:
@danihodovic Btw, we handle a similar situation in repo.open https://github.com/iterative/dvc/blob/master/dvc/repo/__init__.py#L453